nix/default.nix

```diff
@@ -14,14 +14,36 @@
 #  limitations under the License.
 #
 
-{ system ? builtins.currentSystem
-, pkgs ? import <nixpkgs> { inherit system; }
+let
+  nixpkgs-bootstrap = import <nixpkgs> { };
+
+  nixpkgs-16_03 = import (nixpkgs-bootstrap.fetchFromGitHub {
+    owner = "NixOS";
+    repo = "nixpkgs-channels";
+    rev = "baf46b99e33005348fdbd083366c330be4b373f3";
```

Comment:
Branches are moving things, which means `sha256` will be invalid pretty soon (which would break the whole Nix expression). The `rev` was chosen to point to the last commit in the `nixpkgs-channels/nixos-16.03` at the time of writing.